subprojects/docs/src/docs/userguide/authoring-builds/dev_introduction.adoc

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dev_introduction]]
+= Introduction
+
+Build engineers that are ready to start writing their own tasks and plugins should start here.
+
+There are two options to get started developing Gradle scripts:
+
+1. Read the <<gradle_author_intro,introduction chapters>> for a theoretical approach.
+2. Follow the <<author_tutorial,authoring tutorial>> for a hands-on approach.
+
+[[gradle_author_intro]]
+== 1. Introduction to Authoring Gradle
+
+The introduction section goes through the Gradle authoring basics so that you can quickly understand how to create tasks, apply and write plugins, optimize the build, and more.
```

Comment:
It's just a script plugin. Since we won't have any full blow tutorials for the rest of the chapters, they need to write their first plugin in this section.

Reply:
I don't understand your response here.  My point was this isn't really teaching any sort of optimization work, it's just teaching how to write plugins, so we shouldn't include the phrase "optimize the build".

Reply:
Got it. I misunderstood your feedback. I made the change in https://github.com/gradle/gradle/pull/26992